stylesheets/_typography.scss

```diff
@@ -15,6 +15,8 @@
 
 // CORE FONTS - NEW TRANSPORT
 
+// scss-lint:disable NameFormat
```

Comment:
The requirement is that all variables names are lower case and hypenated. For the font stack variables, these have uppercase letters, eg `$NTA-Light` and `$Helvetica-Regular`.

I've disabled the linter to allow these variable names in the font-stack and typography partials.